src/web/screens/AffiliationViews/StorageGridView/components/StorageGridTable.tsx

```diff
@@ -0,0 +1,72 @@
+import { useEffect, useState } from 'react';
+import { TableContainer } from 'web/components/DetailsListUtils/styles';
+import { ScrollablePane } from '@fluentui/react';
+import DetailsList from '@skatteetaten/frontend-components/DetailsList';
+import { initColumns } from '../utils/columns';
+import { Selection, IObjectWithKey } from '@fluentui/react';
+import { getLocalDate } from 'web/utils/date';
+import StatusIcon from '../utils/StatusIcon';
+import { StorageGridObjectArea } from 'web/services/auroraApiClients/storageGridClient/query';
+
+interface Props {
+  filter: string;
```

Comment:
Gi denne et annet navn? Litt forvirrende da andre variabler i denne komponenten også heter filter.

Reply:
Ja ser det kan bli litt forvirrende, endret til currentFilter men åpen for forslag :)